docs/examples/viz_network_animated_billboard.py

```diff
@@ -0,0 +1,274 @@
+
+        if(selected_node is not None):
+            if(selected_node < len(positions)):  # Fix index mismatch bug
+                selected_actor_label.SetPosition(positions[selected_node])
+
+        update_actor(nodes_actor)
+        compute_bounds(nodes_actor)
+
+        compute_bounds(selected_actor_label)
+        update_actor(selected_actor_label)
+        showm.scene.reset_clipping_range()
+        showm.render()
+
+        if counter >= max_iterations:
+            showm.exit()
+    return _timer
+
+###############################################################################
+# Defining interactions with hardware selector
+
+
+scene = window.Scene()
+camera = scene.camera()
+
+selected_node = None
+selected_actor_label = actor.label(
+    "Origin", pos=centers[0], color=(1, 1, 1), scale=(4, 4, 4),)
+shader_to_actor(selected_actor_label, "fragment", "gl_FragDepth = 0;",
+                block="color", debug=True)
+selected_actor_label.PickableOff()
+selected_actor_label.SetCamera(scene.GetActiveCamera())
+lines_actor.PickableOff()
+# nodes_actor.PickableOff()
+
+
+def hovering_callback(obj, event):
+    global selected_node
+    event_pos = showm.iren.GetEventPosition()
+    picking_area = 4
+    res = hsel.Select()
+    hsel.SetArea(event_pos[0]-picking_area, event_pos[1]-picking_area,
+                 event_pos[0]+picking_area, event_pos[1]+picking_area)
+    res = hsel.Select()
+
+    num_nodes = res.GetNumberOfNodes()
+    if (num_nodes < 1):
+        selected_node = None
+    else:
+        sel_node = res.GetNode(0)
+        selected_nodes = set(np.floor(vtknp.vtk_to_numpy(
+            sel_node.GetSelectionList())/2).astype(int))
+        selected_node = list(selected_nodes)[0]
+
+    if(selected_node is not None):
+        if(labels is not None):
+            selected_actor_label.text.SetText(labels[selected_node])
+        else:
+            selected_actor_label.text.SetText("#%d" % selected_node)
+
+        if(selected_node < len(positions)):  # Fix index mismatch bug
+            selected_actor_label.SetPosition(positions[selected_node])
+
+    else:
+        selected_actor_label.text.SetText("")
+    timer_callback(None, None)
+
+###############################################################################
+# We add observers to pick the nodes and enable hardware selector
+
+
+hsel = vtk.vtkHardwareSelector()
+hsel.SetFieldAssociation(vtk.vtkDataObject.FIELD_ASSOCIATION_CELLS)
+hsel.SetRenderer(scene)
+
+###############################################################################
+# All actors need to be added in a scene, so we build one and add our
+# lines_actor and nodes_actor.
+
+scene.add(lines_actor)
+scene.add(nodes_actor)
+scene.add(selected_actor_label)
+
+
+###############################################################################
+# The final step ! Visualize the result of our creation! Also, we need to move
+# the camera a little bit farther from the network. you can increase the
```

Comment:
further? not clear.

Reply:
I've changed it to "farther from the network center", does this help making the sentence clear?